app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -147,7 +147,7 @@ const processVMTasks = vmTasks => {
 
       taskDetails.diskSpaceCompleted = percentComplete * taskDetails.totalDiskSpace;
       taskDetails.diskSpaceCompletedGb = numeral(taskDetails.diskSpaceCompleted).format('0.00b');
-      taskDetails.percentComplete = Math.round(percentComplete * 1000) / 10;
+      taskDetails.percentComplete = Math.round((percentComplete * 1000) / 10);
```

Comment:
The ghost of my former life as a mathematician won't let me leave this be, lol

Can we change this to `Math.round(percentComplete * 100)` 🙏

Reply:
This is reasonable, can totally find reason this friday, promise.